capstone-backend/src/test/java/com/google/sps/BookTest.java

```diff
@@ -0,0 +1,75 @@
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import com.google.sps.data.Book;
+
+/** */
+@RunWith(JUnit4.class)
+public final class BookTest {
```

Comment:
Great job writing comprehensive unit tests - it's good practice to try to have at least one unit test that exercises each public method, which you've done here!

Reply:
Thanks!